src/lib/OpenEXR/ImfCompressor.cpp

```diff
@@ -63,6 +64,7 @@ isValidCompression (Compression c)
         case B44_COMPRESSION:
         case B44A_COMPRESSION:
         case DWAA_COMPRESSION:
+        case ZSTD_COMPRESSION:
         case DWAB_COMPRESSION: return true;
```

Comment:
Don't separate related schemes,
```suggestion
        case DWAA_COMPRESSION:
        case DWAB_COMPRESSION:
        case ZSTD_COMPRESSION: return true;
```